com.github.jkotra.unlockr.yml

```diff
@@ -0,0 +1,36 @@
+app-id: com.github.jkotra.unlockr
+runtime : org.gnome.Platform
+runtime-version : '42'
+sdk : org.gnome.Sdk
+command: unlockr
+
+finish-args:
+  - --share=ipc
+  - --socket=fallback-x11
+  - --socket=wayland
+  - --talk-name=org.freedesktop.Flatpak
+  - --filesystem=host
+  - --persist=path
```

Comment:
not sure this does what you think it does.

Reply:
removed.